src/modules/core/components/ActionsList/ActionsListItem.tsx

```diff
@@ -219,9 +221,21 @@ const ActionsListItem = ({
         onClick={handleSyntheticEvent}
         onKeyPress={handleSyntheticEvent}
       >
-        <div className={styles.avatar}>
+        <div
+          /*
+           * Clicking on UserAvatar would redirect to Actions page and stop
+           * interaction with popover.
+           * stopPropagation prevents event being inherited by child
+           */
+          onClick={stopPropagation}
+          onKeyPress={stopPropagation}
+          role="button"
+          tabIndex={0}
+          className={styles.avatar}
+        >
```

Comment:
I'm good with this change, however there's an `handleOnClick` prop that's coming in, which assumes it can run the click action on the whole item. 

Stopping propagation on the avatar, might impede that to some extent.

But since there's not "proper" way to check the execution requirements of the `onClick` function that's coming in _(well, at least not without going a very deep dark rabbit hole)_, it will have to do for now

Reply:
The onClick was being passed to MemberInfoPopover, so it became clickable (same as the parent listItem), taking the user to the DefaultAction page. 
Preventing the propagation allows MemberInfoPopover to behave as expected.